### PR TITLE
fix(tests): added func tests to fabric8 pipeline.

### PR DESCRIPTION
**Pull request by michaelkleinhenz**

### CODE REVIEW

File: scripts/run-functests.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+docker exec fabric8-planner-builder npm install
+
+docker exec fabric8-planner-builder npm run test:unit
+
+docker exec fabric8-planner-builder npm run build
+
+docker exec  -i fabric8-planner-builder bash -c "cd runtime ; npm install"
+docker exec fabric8-planner-builder bash -c "cd runtime ; npm run test:funcsmoke"
+docker exec  -i fabric8-planner-builder bash -c "cd runtime ; npm run build"
```
Comment (nimishamukherjee) [inline scripts/run-functests.sh]: Do we need to build under the runtime folder?

File: scripts/run-functests.sh [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Build builder image
+cp /tmp/jenkins-env .
+docker build -t fabric8-planner-builder -f deploy/Dockerfile.builder .
+# User root is required to run webdriver-manager update. This shouldn't be a problem for CI containers
+mkdir -p dist && docker run --detach=true --name=fabric8-planner-builder --user=root --cap-add=SYS_ADMIN -e "API_URL=https://api.prod-preview.openshift.io/api/" -e "CI=true" -t -v $(pwd)/dist:/dist:Z fabric8-planner-builder
+
+docker exec fabric8-planner-builder npm install
+
+docker exec fabric8-planner-builder npm run test:unit
+
```
Comment (nimishamukherjee) [inline scripts/run-functests.sh]: Do we need to run unit tests under functional tests?

Comment (michaelkleinhenz) [inline scripts/run-functests.sh]: Good, Point. Will remove them.

File: scripts/run-functests.sh (hunk 1 above)
Comment (michaelkleinhenz) [inline scripts/run-functests.sh]: Yes, we need the runtime for the functional tests as a framework.

Comment (nimishamukherjee) [inline scripts/run-functests.sh]: But in this case we are building the runtime after the functional tests complete.

File: scripts/run-functests.sh [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+docker exec fabric8-planner-builder npm run build
+
+docker exec  -i fabric8-planner-builder bash -c "cd runtime ; npm install"
+docker exec fabric8-planner-builder bash -c "cd runtime ; npm run test:funcsmoke"
+docker exec  -i fabric8-planner-builder bash -c "cd runtime ; npm run build"
+
+docker exec -u root fabric8-planner-builder cp -r /home/fabric8/fabric8-planner/runtime/dist /
```
Comment (nimishamukherjee) [inline scripts/run-functests.sh]: Do we need this step? We will not do anything with the dist folder.

File: scripts/run-functests.sh (hunk 1 above)
Comment (michaelkleinhenz) [inline scripts/run-functests.sh]: ah. Yes. Correct. Weird.

Comment (michaelkleinhenz) [inline scripts/run-functests.sh]: Removed.

File: scripts/run-functests.sh (hunk 2 above)
Comment (michaelkleinhenz) [inline scripts/run-functests.sh]: Done.

File: scripts/run-functests.sh (hunk 3 above)
Comment (michaelkleinhenz) [inline scripts/run-functests.sh]: No, Removed.

**Comment by nimishamukherjee** [comment]
[test]

**Comment by hectorj2f** [comment]
@michaelkleinhenz I was working on a fix here 
https://github.com/fabric8io/fabric8-planner/pull/2003

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1998-fabric8-planner.int.rdu2c.fabric8.io

**Comment by fabric8cd** [comment]
@michaelkleinhenz snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1998-fabric8-planner.int.rdu2c.fabric8.io

**Comment by hectorj2f** [comment]
@michaelkleinhenz I think we can close this PR in favor of https://github.com/fabric8-ui/fabric8-planner/pull/2003

**Comment by hectorj2f** [comment]
Closing this PR! It has been addressed by #2003

**Pull request closed without merging**